pkg/aws/client.go

```diff
@@ -484,6 +509,19 @@ func (c *awsClient) CheckAdminUserNotExisting(userName string) (err error) {
 	return nil
 }
 
+func (c *awsClient) CheckAdminUserExists(userName string) (err error) {
+	userList, err := c.iamClient.ListUsers(&iam.ListUsersInput{})
```

Comment:
Can we use `GetUser` instead and fetch the actual user name directly? (I'm wondering if there's a reason, the old code also uses `ListUsers` instead of `GetUser`...)